src/seaborn_image/_grid.py

```diff
@@ -67,6 +67,10 @@ class ImageGrid:
         Maximum data value that colormap covers, by default None
     interpolation : str, optional
         `matplotlib.pyplot.imshow` interpolation method used, by default None
+    norm : `matplotlib.colors.Normalize`, optional
+        `matplotlib` Normalize instance used to scale scalar data before
+        mapping to colors using cmap.
+        For the moment being the same normalization is used for all images in the grid.
```

Comment:
Since `ImageGrid` supports input images as a list of 2D images, we definitely want to support different norm values for different images (similar to different cmap)

Reply:
Yeah, it shouldn't be too difficult to implement, I'll make another commit on my branch in a moment.

Reply:
I think that's it, if you like the new changes i think everything should be ready to be merged.